letsencrypt-auto-source/Dockerfile.centos6

```diff
@@ -30,6 +30,11 @@ RUN update-ca-trust
 # Copy code:
 COPY . /home/lea/certbot/letsencrypt-auto-source
 
+# Tweak uname binary for tests on fake 32bits
+COPY uname_wrapper.sh /bin
```

Comment:
It looks like this path needs to be updated to since the file was moved under `tests`.

Reply:
Fixed.